test/distributed/tensor/test_dtensor_export.py

```diff
@@ -58,7 +70,21 @@ def graph_capture_and_aot_export_joint_with_descriptors(model, inputs):
     with torch._dynamo.config.patch(install_free_tensors=True):
         # TODO: switch to use the official graph_capture API once it is ready
         gm = _dynamo_graph_capture_for_export(model)(inputs)
-    return aot_export_joint_with_descriptors_alone(gm, inputs)
+    
+    # TODO we actually need to get this from graph capture output, but this is fine.
+    fake_vals: list[torch.Tensor] = [] 
```

Comment:
Don't want to share code with export here because export one is also  grabbing fake mode from inputs instead of GraphCaptureOutput.